### PR TITLE
Add an example directory for building simulation images

### PR DESCRIPTION
**Pull request by mikebauer**
## Description of change
Add an example directory with a README.md describing how to build and locally run a simulation docker image.

## Guide to reproduce test results.
Build and run the image as per the instructions:
```
./resim/examples/sim_runner/build.sh
./resim/examples/sim_runner/test_run.sh
```

## Checklist:
- [x] I have self-reviewed this change.
- [x] I have tested this change.
- [ ] This change is covered by tests that are already landed, or in this PR.

### CODE REVIEW

File: resim/examples/sim_runner/test_run.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#!/bin/bash
+
+set -e
+
+SIM_RUNNER_DIR=resim/examples/sim_runner
+
+# Set up the experience in local folders in a volume which the dev container #
```
Comment (knowlsie) [inline resim/examples/sim_runner/test_run.sh:8]:
```suggestion
# Set up the experience in local folders in a volume which the dev container
```

File: resim/examples/sim_runner/test_run.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# mounts at /root/.
+
+INPUTS=/root/inputs
+OUTPUTS=/root/outputs
+
+mkdir -p "${INPUTS}" "${OUTPUTS}"
+cp "${SIM_RUNNER_DIR}/experience.sim" "${INPUTS}"
+
+# The ReRun worker will run a command similar to this, ensuring that
```
Comment (knowlsie) [inline resim/examples/sim_runner/test_run.sh:17]: This comment isn't totally clear to me

File: resim/examples/sim_runner/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Creating a Sim Runner Container
+
+## Introduction
+The contents of this folder are intended to demonstrate how to create a docker
+image that can be run by ReRun. In order to be compatible with ReRun, this
+docker image should exepct its inputs in a directory to be mounted at
```
Comment (knowlsie) [inline resim/examples/sim_runner/README.md:6]:
```suggestion
docker image should expect its inputs in a directory to be mounted at
```

File: resim/examples/sim_runner/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+with its logs and other outputs. To get started, we'll be using the `resim_run`
+binary as our representative simulator to be run in ReRun. Since we want to test
+this runner image out locally, we will use a test script `test_run.sh` to stand
+in for ReRun.
+
+## Prerequisites
+
+To run this example, it is assumed that you have cloned the `resim-ai/open-core`
+repository and that you are operating inside of the development docker container
+shipped with it. See the [getting started guide](https://docs.resim.ai/) for
```
Comment (knowlsie) [inline resim/examples/sim_runner/README.md:17]:
```suggestion
shipped with it. See the [getting started guide](https://docs.resim.ai/#getting-started) for
```

File: resim/examples/sim_runner/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+more information.
+
+## Running the example
+
+The steps for running this example are as follows:
+
+ - Clone and navigate into to the `open-core` repository.
```
Comment (knowlsie) [inline resim/examples/sim_runner/README.md:24]:
```suggestion
 - Clone and navigate into the `open-core` repository.
```

**Review by knowlsie — APPROVED** [review]
LGTM, some nits!

File: resim/examples/sim_runner/test_run.sh (hunk 1 above)
Comment (mikebauer) [reply resim/examples/sim_runner/test_run.sh:8]: Good catch. Fixing.

File: resim/examples/sim_runner/test_run.sh (hunk 2 above)
Comment (mikebauer) [reply resim/examples/sim_runner/test_run.sh:17]: Okay. I've tried to re-word this slightly to be more clear.

File: resim/examples/sim_runner/README.md (hunk 3 above)
Comment (mikebauer) [reply resim/examples/sim_runner/README.md:6]: Good catch. Fixed.

File: resim/examples/sim_runner/README.md (hunk 4 above)
Comment (mikebauer) [reply resim/examples/sim_runner/README.md:17]: Good call. I've added this.

File: resim/examples/sim_runner/README.md (hunk 5 above)
Comment (mikebauer) [reply resim/examples/sim_runner/README.md:24]: Good catch. Fixed.

**Pull request merged**